lib/livebook/application.ex

```diff
@@ -6,8 +6,16 @@ defmodule Livebook.Application do
     setup_optional_dependencies()
     ensure_directories!()
     set_local_file_system!()
-    ensure_distribution!()
-    validate_hostname_resolution!()
+
+    if Livebook.Config.boolean!("LIVEBOOK_EPMDLESS", false) do
```

Comment:
I think for consistency we should parse this in `lib/livebook.ex` and put in config.

Reply:
I was thinking about this... but the app env var is not really used anywhere else and I wanted to avoid us reading it (we should use Livebook.EPMD.dist_port instead to detect the status). Do you prefer the app env var anyway?

Reply:
I would put it in config, even if used in this single place. I think we `env -> config -> access` almost everywhere and that makes it more centralized :)